content/en/synthetics/browser_tests/_index.md

```diff
@@ -41,25 +41,30 @@ Define the configuration of your browser test.
-6. **How often should Datadog run the test?** Intervals are available between every 15 minutes to once per week. [Contact support][4] to enable additional frequencies for your test.
+4. **Devices**: The devices to run your test on. Available devices are `Laptop Large`, `Tablet`, and `Mobile Small`.
+5. **Locations**: The Datadog managed locations to run the test from. Many AWS locations from around the world are available. You can also set up a [Private Location][2] to run a Synthetics Browser test on a private URL not accessible from the public internet.
+6. **How often should Datadog run the test?** Intervals are available between every 5 minutes to once per week.
 
 ### Use global variables
 
-You can use the [global variables defined in the `Settings`][5] in the URL, as well as in the Advanced Options of your browser tests. To display your list of variables, type `{{` in your desired field.
+You can use the [global variables defined in the **settings**][3] in the URL, as well as in the Advanced Options of your browser tests. To display your list of variables, type `{{` in your desired field.
 
 {{< img src="synthetics/browser_tests/using_variables_browser.mp4" alt="Using Variables in Browser Tests" video="true"  width="80%" >}}
 
-### Alert conditions
+### Define alert conditions
 
-An alert is triggered if any assertion fails for `<INSERT_NUMBER>` minutes from any `<INSERT_NUMBER>` of `<NUMBER_OF_CHOSEN>` locations.
+You can customize alert conditions to define the circumstances under which you want a test to send a notification alert.
+
+* An alert is triggered if any assertion fails for `<INSERT_NUMBER>` minutes from any `<INSERT_NUMBER>` of `<NUMBER_OF_CHOSEN>` locations. This alerting rule allows you to specify for how much time and on how many locations a test needs to fail before triggering the notification using. 
+* Retry `<INSERT_NUMBER>` times before location is marked as failed. This allows you to define how many consecutive test failures need to happen for a location to be considered as failed. By default, there is a 300ms wait before retrying a test that failed. This interval can be configured via the [API][4].
+
+{{< img src="synthetics/browser_tests/alerting_rules.png" alt="Browser test alerting rule"  >}}
 
 ### Notify your team
 
 A notification is sent according to the set of alerting conditions. To configure your notifications:
 
-1. Enter a **message** for the browser test. This field allows standard [Markdown formatting][6] and supports the following [conditional variables][7]:
+1. Enter a **message** for the browser test. This field allows standard [Markdown formatting][5] and supports the following [conditional variables][6]:
```

Comment:
@MargotLepizzera  Is this the correct link? Looks like it links out the integrations page, which fits with the content below in 2.

Reply:
@sarina-dd great catch! This should link to monitor pages.

- conditional variables to https://docs.datadoghq.com/monitors/notifications/?tab=is_alert#integrations
- services to https://docs.datadoghq.com/monitors/notifications/?tab=is_alert#integrations